extmod/network_ninaw10.c

```diff
@@ -217,7 +217,7 @@ static mp_obj_t network_ninaw10_active(size_t n_args, const mp_obj_t *args) {
 
             if (fwver < fwmin) {
                 mp_raise_msg_varg(&mp_type_OSError,
-                    MP_ERROR_TEXT("Firmware version mismatch. Minimum supported firmware is v%d.%d.%d found v%d.%d.%d\n"),
+                    MP_ERROR_TEXT("firmware version mismatch. Minimum supported firmware is v%d.%d.%d found v%d.%d.%d\n"),
```

Comment:
I suggest "firmware version mismatch, minimum supported firmware is v%d.%d.%d found v%d.%d.%d" (ie no full stop, no trailing newline).

Reply:
Okay.